src/main/java/org/fundacionjala/coding/omar/movies/Customer.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.omar.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Omar Limbert Huanca Sanchez AT06.
+ */
+public class Customer {
+    private static final int ZERO = 0;
+    private static final String NEW_LINE = "\n";
+    private static final int CAPACITY = 100;
+    private final String customerName;
+    private final List<Rental> rentalsList = new ArrayList<>();
+
+    /**
+     * Constructor for Customer.
+     *
+     * @param name customer name.
+     */
+    public Customer(String name) {
+        this.customerName = name;
+    }
+
+    /**
+     * @param rental this method add rental to rental list.
+     */
+    public void addRental(Rental rental) {
+        rentalsList.add(rental);
+    }
+
+    /**
+     * This Method returns, Rental Overview.
+     *
+     * @return StringBuilder.
+     */
+    public String getRentalOverview() {
+        StringBuilder overviewBuilder = new StringBuilder(CAPACITY);
+
+        overviewBuilder.append(String.format("%s %s", "Customer:", customerName))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "%s %s" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)